src/config/app.ts

```diff
@@ -111,10 +110,10 @@ function discoverPhpVersionsForJob(job: JobDefinitionFromFile, config: Config):
 
 function discoverComposerDependencySetsForJob(job: JobDefinitionFromFile, config: Config): ComposerDependencySet[] {
     const dependencySetFromConfig = job.dependencies
-        ?? (config.lockedDependenciesExists ? ComposerDependencySet.LOCKED : ComposerDependencySet.LATEST);
+        ?? (config.lockedDependenciesExists ? ComposerDependencySet.LOCKED : ComposerDependencySet.LOWEST);
 
     if (isAnyComposerDependencySet(dependencySetFromConfig)) {
-        return [ ComposerDependencySet.LOWEST, ComposerDependencySet.LATEST ];
+        return [ ComposerDependencySet.LOWEST ];
```

Comment:
I wonder if we should keep this as it is explicitly required from a manually written config entry. If some component does not need/want `latest`, they can remove that from their config.